### ISSUE TITLE
How can I get my own training data

### ISSUE THREAD

**Issue by LXYTSOS** [eragonruan/text-detection-ctpn#76]
I want to  prepare my own training data, after reading split_label.py, I don't know how to get the txt files in the label path ready, could you tell how to prepare those txt files?

**Comment by walsvid** [comment]
Those txt files are format like: `x1,y1,x2,y2,x3,y3,x4,y4,language,content` e.g. `230,182,310,180,310,205,230,205,Latin,Chicken`. Please check [this](http://rrc.cvc.uab.es/?ch=8&com=tasks) website and then register a account, download the dataset `icdar2017, multi lingual scene dataset`. 

If you would like to use your own data, you can convert your own data format to the ground truth txt file which mentioned before, and then you can use `split_label.py` and `ToVoc.py` transform it to VOC format.

**Comment by LXYTSOS** [comment]
(x1,y1)--upper left，(x2,y2)--upper right，(x3,y3)--lower left，(x4,y4)--lower right. Am I right?

**Comment by rainman0709** [comment]
I guess, it's A text:(x1,y1)--upper left，(x2,y2)--lower right， B text:(x3,y3)--upper left，(x4,y4)--lower right，A text，B text

**Comment by walsvid** [comment]
I'm afraid not, the order of the coordinates is clockwise, (x3,y3)--**lower right**，(x4,y4)--**lower left**. You can check the value of the example`230,182,310,180,310,205,230,205,Latin,Chicken`.

**Comment by eragonruan** [comment]
@LXYTSOS walsvid is right, the order is clockwise.

**Comment by forever1love0612** [comment]
Do I need to do some preprocessing to my images in the image path before I run split_label.py? @eragonruan
